light_sdk/src/tx_builders/mod.rs

```diff
@@ -0,0 +1,65 @@
+use borsh::BorshSerialize;
+use namada_core::ledger::governance::storage::proposal::ProposalType;
+use namada_core::proto::Section;
+use namada_core::proto::SignatureIndex;
+use namada_core::proto::Signer;
+use namada_core::proto::TxError;
+use namada_core::proto::{Signature, Tx};
+use namada_core::types::account::AccountPublicKeysMap;
+use namada_core::types::address::Address;
+use namada_core::types::chain::ChainId;
+use namada_core::types::dec::Dec;
+use namada_core::types::hash::Hash;
+use namada_core::types::key::{common, secp256k1};
+use namada_core::types::storage::Epoch;
+use namada_core::types::time::DateTimeUtc;
+use namada_core::types::token;
+use namada_core::types::token::{Amount, DenominatedAmount, MaspDenom};
+use namada_core::types::transaction::Fee;
+use namada_core::types::transaction::GasLimit;
+use std::collections::BTreeMap;
+use std::str::FromStr;
+
+pub mod pos;
+
+pub(in crate::tx_builders) fn build_tx(
```

Comment:
no generics - everything should be passed in as strings

Reply:
This function is not exposed to the SDK user